client/components/FishList.jsx

```diff
@@ -26,16 +39,62 @@ class FishList extends React.Component {
   }
 }
 
+function sortingAZ (items) {
+  items.sort((a, b) => {
+    let nameA = a.name.toUpperCase()
+    let nameB = b.name.toUpperCase()
+    if (nameA < nameB) {
+      return -1
+    }
+    if (nameA > nameB) {
+      return 1
+    }
+    return 0
+  })
+}
+
+function sortingZA (items) {
+  items.sort((a, b) => {
+    let nameA = a.name.toUpperCase()
+    let nameB = b.name.toUpperCase()
+    if (nameA > nameB) {
+      return -1
+    }
+    if (nameA < nameB) {
+      return 1
+    }
+    return 0
+  })
+}
+
 const mapStateToProps = state => {
-  return {
-    fish: state.fish,
-    info: state.info
+  if (state.sort.sortOrder == 'AZ') {
+    sortingAZ(state.fish)
+    return {
+      fish: state.fish,
+      info: state.info,
+      sortOrder: state.sort.sortOrder
+    }
+  } else if (state.sort.sortOrder == 'ZA') {
+    sortingZA(state.fish)
+    return {
```

Comment:
Lines 80-84 are identical to lines 73-77. Try refactoring this a little so that you don't need to write the same code twice :-)